### PR TITLE
fix(schemas): adding risk evaluation plugin output to context schema:

### PR DESCRIPTION
**Pull request by alonp99**
## Release Notes

- **New Features**
	- Introduced several dependency updates across multiple packages to enhance performance and compatibility.

- **Bug Fixes**
	- Updated dependencies for improved stability and security, including:
		- `@ballerine/common` to version 0.9.33
		- `@ballerine/workflow-core` to version 0.6.44
		- `@ballerine/react-pdf-toolkit` to version 1.2.33
		- Various other package updates.

- **Documentation**
	- Changelog entries have been updated to reflect the latest version changes for clarity.

### CODE REVIEW

**Review by tomer-shvadron — APPROVED** [review]

**Review by Omri-Levy — APPROVED** [review]

File: packages/common/src/schemas/documents/schemas/risk-evaluation-plugin-schema.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+              rules: Type.Optional(
+                Type.Array(
+                  Type.Object({
+                    key: Type.Optional(Type.String()),
+                    value: Type.Optional(Type.Array(Type.String())),
+                    operator: Type.Optional(Type.String()),
+                  }),
+                ),
+              ),
+              operator: Type.Optional(Type.String()),
```
Comment (liorzam) [inline packages/common/src/schemas/documents/schemas/risk-evaluation-plugin-schema.ts:37]: What is this schema related to ?
